admin/RebuildIndexesUsingCollations.pl

```diff
@@ -3,10 +3,35 @@
 use warnings;
 
 use FindBin;
+use Getopt::Long qw( GetOptions );
 use lib "$FindBin::Bin/../lib";
 use MusicBrainz::Server::Context;
+use aliased 'MusicBrainz::Server::DatabaseConnectionFactory' => 'Databases';
 
-my $c = MusicBrainz::Server::Context->create_script_context(database => 'MAINTENANCE');
+my $database = 'MAINTENANCE';
+my $concurrently = 1;
+my $show_help = 0;
+
+sub usage {
+    print <<~EOF;
+        Usage: RebuildIndexesUsingCollations.pl [options]
+
+            --database          database to connect to (default: MAINTENANCE)
+            --[no]concurrently  enable or disable concurrent reindexing
+                                (default: enabled)
```

Comment:
Same for this option:

```suggestion
            --[no]concurrently  enable or disable concurrent reindexing
                                (concurrent is slower but doesn't lock any table)
                                (default: enabled)
```